lib/Sema/LookupVisibleDecls.cpp

```diff
@@ -839,6 +886,14 @@ static void lookupVisibleMemberDecls(
   lookupVisibleMemberDeclsImpl(BaseTy, overrideConsumer, CurrDC, LS, Reason,
                                TypeResolver, GSB, Visited);
 
+  llvm::DenseSet<DeclBaseName> knownMembers;
+  for (auto &kv : overrideConsumer.FoundDecls) {
+    knownMembers.insert(kv.first);
```

Comment:
Could we avoid this copy if unless it's `hasDynamicMemberLookupAttribute()`?

Reply:
Good idea.

Reply:
Fixed in my latest changes.  Also added the check for `mayHaveMembers`.